mllib/src/main/scala/org/apache/spark/mllib/clustering/KMeansModel.scala

```diff
@@ -77,6 +77,22 @@ class KMeansModel @Since("1.1.0") (@Since("1.0.0") val clusterCenters: Array[Vec
   def predict(points: JavaRDD[Vector]): JavaRDD[java.lang.Integer] =
     predict(points.rdd).toJavaRDD().asInstanceOf[JavaRDD[java.lang.Integer]]
 
+  /**  */
+  def distanceToCenters(point: Vector): (Int, Double) = {
```

Comment:
Indeed it was... Sorry about that.

2015-09-21 14:01 GMT-07:00 Bertrand Dechoux <notifications@github.com>:

> In
> mllib/src/main/scala/org/apache/spark/mllib/clustering/KMeansModel.scala
> <https://github.com/apache/spark/pull/8849#discussion_r40025294>:
>
> > @@ -77,6 +77,22 @@ class KMeansModel @Since("1.1.0") (@Since("1.0.0") val clusterCenters: Array[Vec
> >    def predict(points: JavaRDD[Vector]): JavaRDD[java.lang.Integer] =
> >      predict(points.rdd).toJavaRDD().asInstanceOf[JavaRDD[java.lang.Integer]]
> >
> > +  /**  */
> > +  def distanceToCenters(point: Vector): (Int, Double) = {
>
> Indeed not. The diff I received must have been a work in progress. This is
> now fixed.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/spark/pull/8849/files#r40025294>.
>



-- 
Timothée Rebours
13, rue Georges Bizet
78380 BOUGIVAL